bin/good-first-issue.js

```diff
@@ -25,22 +26,30 @@ cli
     if (!(input in projects)) {
       console.log(chalk.red(`"${input}" was not found in good-first-issue.`))
       console.log('--------------------------------')
-      console.log('If you\'d like to add a new project to good-first-issue,')
-      console.log('please see the module\'s Github repository: ' + chalk.cyan('https://github.com/bnb/good-first-issue#adding-new-projects'))
+      console.log("If you'd like to add a new project to good-first-issue,")
+      console.log(
+        "please see the module's Github repository: " +
+          chalk.cyan(
+            'https://github.com/bnb/good-first-issue#adding-new-projects'
+          )
+      )
+      process.exit(0)
+    }
+
+    const issues = await search(projects[input].q)
+
+    if (issues.length === 0) {
+      console.log(chalk.yellow(`No Good First Issues were found in ${input}`))
       process.exit(0)
     }
-    search(projects[input].q, (error, issues) => {
-      if(error) throw error
-      
-      // Call the log functionality, output the result to the console.
-      log(issues, projects[input].name, function(error, output) {
-        if(error) throw error
-        // Configure the randomizer for the pool of good-first-issues. This cannot exceed how many entries are actually available from the API.
-        var random = Math.floor(Math.random() * Math.floor(output.length - 1));
-    
-        // Log the issue!
-        console.log(output[random].toString())
-      })
-    })    
+    // Call the log functionality, output the result to the console.
+    log(issues, projects[input].name, function(error, output) {
```

Comment:
if we are doing async in search
i suggest we should do `log` also async :smile: 
/cc @bnb

Reply:
Not sure what that would look like. Care to give an example @maddhruv?

I have no opinion one way or the other at present. Perhaps a good change for another PR.

Reply:
Ah, looking deeper into the code that makes sense. +1.

Reply:
Looking deeper, that makes sense.

Reply:
So i'm not sure anymore what we are going to do? We go for normal Promises (node 4) or we leave it async await (node 8)?

Reply:
Leave it async/await (node 8) since the module already won't work with Node.js v6. I was planning on adding Node.js 8 to the package.json as the value for engines anyway :)